lib/util/buffer.js

```diff
@@ -170,6 +170,49 @@ module.exports = {
   hexToBuffer: function hexToBuffer(string) {
     assert(js.isHexa(string));
     return new buffer.Buffer(string, 'hex');
+  },
+
+  /**
+   * Browser shim for buffer compare
+   *
+   * MIT. Copyright by Feross Aboukhadijeh, and other contributors.
+   * Originally forked from an MIT-licensed module by Romain Beauxis.
+   * @see {https://github.com/feross/buffer/blob/bc33ea4618e6846148e35aee78dfc28d999bdd1c/index.js#L264}
+   */
```

Comment:
Considering that it's mostly to support Node.js v0.10, it may be okay to have this inlined. However I do agree, we should have tests that cover it.